src/rfkill/rfkill.c

```diff
@@ -35,9 +35,22 @@
 #include "string-util.h"
 #include "udev-util.h"
 #include "util.h"
+#include "list.h"
 
+/* Note that any write is delayed until exit and the rfkill state will not be
+ * stored for rfkill indices that disappear after a change. */
 #define EXIT_USEC (5 * USEC_PER_SEC)
 
+typedef struct write_queue_item {
+        LIST_FIELDS(struct write_queue_item, queue);
+        int rfkill_idx;
+        char *file;
+        int state;
+} write_queue_item;
+
+static LIST_HEAD(write_queue_item, write_queue);
```

Comment:
no, unnecessary global variables please. Doesn't matter much, but I think it would be nicer to avoid this, and just make it a local varibale of main() or so, which is passed through as pointer... dunno.